src/middleware/validation.schema.js

```diff
@@ -26,6 +26,25 @@ const followOrUnfollow = {
     followeeId
   }
 };
+const title = Joi.string()
```

Comment:
Good job so far @tobechukwuobitube 
I think there should be no minimum for the number of text for the `title`, `description`, `content`, or `image`. What happens if it is a draft and the user does not have all this information yet. Is there any particular reason why you enforced this validation?

Reply:
Validations are made so that the user input will conform to the requirements of the application. Give an example where a user is supplying a 2-letter word for a `title`, or `description` or `content`, or a situation where we are using an image platform that should return a string as a `url` but rather returns a 2-letter string, the validations are there to check these use cases.